<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrendGraphWidget</class>
 <widget class="QWidget" name="TrendGraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="PlotFrameTemperature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetShowControlsTemperature" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSelectMinTimeTemperature">
          <property name="text">
           <string>Set Min Time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="timeEditSelectMinTimeTemperature">
          <property name="styleSheet">
           <string notr="true">QTimeEdit{
     font-size: 16px;
     background: rgb(249, 247, 248);
     border: 1px solid rgb(218, 220, 224);
     min-height: 40px;
     max-height: 40px;
     padding-right: 2px;
     min-width: 200px;
}

QTimeEdit::up-button{
    image: url(:/images/plus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: right;
    padding: 0px;
}

QTimeEdit::down-button{
   image: url(:/images/minus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: left;
    padding: 0px;
}

QTimeEdit::up-down{
image: url(:/images/plus-button.svg);
}

QTimeEdit::text{
   color: red;
    background-color: blue;
    padding: 4px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>133</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonShowFullRangeTemperature">
        <property name="text">
         <string>Show Full Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSelectMaxTimeTemperature">
          <property name="text">
           <string>Set Max Time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="timeEditSelectMaxTimeTemperature">
          <property name="styleSheet">
           <string notr="true">QTimeEdit{
     font-size: 16px;
     background: rgb(249, 247, 248);
     border: 1px solid rgb(218, 220, 224);
     min-height: 40px;
     max-height: 40px;
     padding-right: 2px;
     min-width: 200px;
}

QTimeEdit::up-button{
    image: url(:/images/plus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: right;
    padding: 0px;
}

QTimeEdit::down-button{
   image: url(:/images/minus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: left;
    padding: 0px;
}

QTimeEdit::up-down{
image: url(:/images/plus-button.svg);
}

QTimeEdit::text{
   color: red;
    background-color: blue;
    padding: 4px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="1" column="4">
        <widget class="QSlider" name="verticalSliderLiquidMinYValue">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSlider" name="verticalSliderLiquidMaxYValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QFrame" name="PlotFrameLiquid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widgetShowControlsLiquid" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="4">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QTimeEdit" name="timeEditSelectMaxTimeLiquid">
          <property name="styleSheet">
           <string notr="true">QTimeEdit{
     font-size: 16px;
     background: rgb(249, 247, 248);
     border: 1px solid rgb(218, 220, 224);
     min-height: 40px;
     max-height: 40px;
     padding-right: 2px;
     min-width: 200px;
}

QTimeEdit::up-button{
    image: url(:/images/plus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: right;
    padding: 0px;
}

QTimeEdit::down-button{
   image: url(:/images/minus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: left;
    padding: 0px;
}

QTimeEdit::up-down{
image: url(:/images/plus-button.svg);
}

QTimeEdit::text{
   color: red;
    background-color: blue;
    padding: 4px;
}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelSelectMaxTimeLiquid">
          <property name="text">
           <string>Set Max Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>319</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonShowFullRange">
        <property name="text">
         <string>Show Full Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>319</width>
          <height>41</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelSelectMinTimeLiquid">
          <property name="text">
           <string>Set Min Time</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="timeEditSelectMinTimeLiquid">
          <property name="styleSheet">
           <string notr="true">QTimeEdit{
     font-size: 16px;
     background: rgb(249, 247, 248);
     border: 1px solid rgb(218, 220, 224);
     min-height: 40px;
     max-height: 40px;
     padding-right: 2px;
     min-width: 200px;
}

QTimeEdit::up-button{
    image: url(:/images/plus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: right;
    padding: 0px;
}

QTimeEdit::down-button{
   image: url(:/images/minus-button.svg);
    background-color: rgb(64, 77, 83);
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    subcontrol-origin: padding;
    subcontrol-position: left;
    padding: 0px;
}

QTimeEdit::up-down{
image: url(:/images/plus-button.svg);
}

QTimeEdit::text{
   color: red;
    background-color: blue;
    padding: 4px;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widgetControlsTimeWindow" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>238</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>208</width>
            <height>14</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButtonPlusTimeWindow">
            <property name="styleSheet">
             <string notr="true">QPushButton{
   background-color: rgb(64, 77, 83);
   color: rgb(249, 247, 248 );
   min-height: 40px;
 min-width: 40px;
}
QPushButton:pressed {
     border: 1px solid white;    
 }</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonMinusTimeWindow">
            <property name="styleSheet">
             <string notr="true">QPushButton{
   background-color: rgb(64, 77, 83);
   color: rgb(249, 247, 248 );
   min-height: 40px;
 min-width: 40px;
}
QPushButton:pressed {
     border: 1px solid white;    
 }
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QTextEdit" name="textEditTimeWindow">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>42</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>42</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SiemensHMI.qrc"/>
 </resources>
 <connections/>
</ui>
